### PR TITLE
Master

### PR DESCRIPTION
**Pull request by mjfrb**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getOperations.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Subscription/createOperation/action",
```
Comment (wilcobmsft) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getOperations.json]: "createOperation"? That doesn't seem right?

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}/providers/Microsoft.Subscription/createSubscription": {
```
Comment (wilcobmsft) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json]: I wish we were more consistent with "name" and "id" in our swaggers. This should ideally be billingAccountName and invoiceSectionName.

**Review by wilcobmsft — CHANGES_REQUESTED** [review]

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getSubscriptionOperation.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "operationId": "e4b8d068-f574-462a-a76f-6fa0afc613c9",
+    "api-version": "2018-11-01-preview"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "/providers/Microsoft.Subscription/subscriptionOperations/e4b8d068-f574-462a-a76f-6fa0afc613c9?api-version=2018-11-01-preview",
+        "Retry-After": "60"
```
Comment (wilcobmsft) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getSubscriptionOperation.json]: Retry-After value needs to be an integer, not a string.

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+          "202": {
+            "description": "Accepted. Subscription update is in progress.",
+            "headers": {
+              "Location": {
+                "description": "The URL where the status of the asynchronous operation can be checked.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "string"
```
Comment (wilcobmsft) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json]: integer, not string. From https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html: "The value of this field can be either an HTTP-date or an integer number of seconds (in decimal) after the time of the response."

**Review by wilcobmsft — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@mjfrb has this API been reviewed by ARM yet?

**Comment by mjfrb** [comment]
Yes, this needs to be reviewed by ARM, I've sent mail requesting this

**Comment by ravbhatnagar** [comment]
This was reviewed in person by ARM. Signing off from ARM side. @mjfrb - Just one thing, you dont need to model the API for tracking the status of long running operation in swagger. It comes from the header. But removing may mean a breaking change.

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getOperations.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Subscription/createSubscription/action",
+            "display": {
+              "provider": "Microsoft Subscription",
+              "resource": "Resources",
+              "operation": "Create a subscription",
+              "description": "Create an Azure subscription."
```
Comment (jhendrixMSFT) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples/getOperations.json]: The description isn't part of the schema and is causing validation errors, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/455086153).

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**